src/server/game/Entities/Player/Player.h

```diff
@@ -1695,6 +1695,47 @@ class TC_GAME_API Player : public Unit, public GridObject<Player>
         void BuildPlayerRepop();
         void RepopAtGraveyard();
 
+        /*********************************************************/
+        /***                  ANTICHEAT SYSTEM                   ***/
+        /*********************************************************///
+        // AntiCheat
```

Comment:
It would be better to move all the anticheat stuff to its own class and call it from Player